media/pages/sk/ochrana-osobnych-udajov/page.html

```diff
@@ -3,10 +3,10 @@
 <h1>Ochrana osobných údajov</h1>
  
 
-<p>
-INFORMÁCIA PREVÁDZKOVATEĽA, POUČENIE O PRÁVACH DOTKNUTEJ OSOBY A SÚHLAS SO SPRACOVANÍM OSOBNÝCH
-ÚDAJOV
-</p>
```

Comment:
Tento jeden nadpis daj pls `h2`. Kedze je to nadradeny nadpis ostatnym nadpisom nizsie.